components/ModalManager/ModalManager.tsx

```diff
@@ -0,0 +1,71 @@
+import { createStyles, CSSObject, Modal as WindowModal } from '@mantine/core';
+import React, { useContext } from 'react';
+import CatModalView from '../CatModal/CatModalView';
+import InterestZoneAdd from '../InterestZoneAdd/InterestZoneAdd';
+import InterestZoneView from '../InterestZoneView/InterestZoneView';
+import { ModalContext } from '../Providers/ModalProvider';
+
+import { InterestZoneProviderContext } from '../Providers/ZoneProvider';
+import CatModalHeader from '../CatModal/CatModalHeader';
+import ZoneModalHeader from './InterestZoneModalHeader';
+
+const useStyles = createStyles((theme) => ({
+  manager: {
+    '& .mantine-Modal-modal': {
```

Comment:
No changes required, just a question. What does the `&` character do in this context?